packages/grid/_modules_/grid/hooks/features/columnMenu/useGridColumnMenu.ts

```diff
@@ -9,7 +9,7 @@ export const useGridColumnMenu = (apiRef: GridApiRef): void => {
   const [gridState, setGridState, forceUpdate] = useGridState(apiRef);
 
   const showColumnMenu = React.useCallback(
-    (field: string, id: string, labelledby: string) => {
+    (field: string, id?: string, labelledby?: string) => {
```

Comment:
I maintain my point from my previous PR, this `labelledBy` should not go here.
and type should be `showColumnMenu(id: GridRowId, field: string)`
Api methods on apiRef should just take the min params

Reply:
I agree we need a better solution for this. I did it so it doesn't block the development of this feature but still supports the other one. If possible I would prefer to ignore it for now and tackle that problem separetly.

Reply:
created -> https://github.com/mui-org/material-ui-x/issues/1365